content/messages/messages-de.yaml

```diff
@@ -0,0 +1,116 @@
+build_up_your_catalog: "Build up your catalog"
+build_up_your_catalog_description: "Having a catalog modelling that is totally tailored to your end-users needs, is really crucial. Before beginning to play with Akeneo, be sure to understand how these few key settings work."
+users_and_permissions: "Users and permissions"
+users_and_permissions_description: "As an administrator, you will need to manage all the users that will access the PIM. Here are some articles to help you understand how to handle the users and their permissions."
+languages_and_translations: "Languages and translations"
+languages_and_translations_description: "We love being international here. The more languages the PIM will speak, the happier we will be. There is already a lot of happiness at Akeneo, but let's bring some more!"
+adminstration_matters: "Administration matters"
+adminstration_matters_description: "This is a kind of 'Miscellaneous' theme: you will need these articles to grasp some PIM concepts, but as you will see below, these concepts are totally unrelated."
+your_first_steps_in_akeneo: "Your first steps into Akeneo"
+your_first_steps_in_akeneo_description: "You are very new to Akeneo and you don't know where to begin. Here are a few articles to help you understand the basics. Enjoy!"
+take_the_power_over_your_products: "Take the power over your products"
+take_the_power_over_your_products_description: "Everything you always wanted to know about the products enrichment inside Akeneo. Brace yourself, we've brought some articles just for you."
+boost_your_productivity: "Boost your productivity"
+boost_your_productivity_description: "You are a PIM master and you know how to perfectly handle your products inside Akeneo. But still, you think you can do better. Check out these awesome features that will help you be so much more productive."
+collaborative_work: "Collaborative work"
+collaborative_work_description: "At Akeneo, we think working as a team is the only way to succeed. You should be able to do the same thanks to your PIM. Empower your coworkers, enrich your products as a team!"
+import_export_data_in_your_pim: "Import/export data in your PIM"
+import_export_data_in_your_pim_description: "Anything you need to know about importing data inside your dearest PIM. You'll also find here infomartions about exporting data. Cause yeah, products information you cannot export is kind of useless, isn't it?"
+manage_your_images_assets: "Manages your images, files, assets"
+manage_your_images_assets_description: "Product information without any asset, it's like Akeneo without Ziggy, it's depressing. So here is how to handle these assets that will liven up your products' sheets."
+
+only_available_in_our_EE: "Only available in our"
+entreprise_edition: "Entreprise Edition"
+summary: "Summary"
+take_a_look_at_these_related_articles: "Be curious! Take a look at these related articles"
+
+end_of_the_internet: "Oops! You'v reached the end of the Internet!"
```

Comment:
You've

Reply:
The English translation in this file will soon be replaced by German ones. ;)